src/riak_kv_wm_utils.erl

```diff
@@ -226,12 +226,18 @@ encode_value(V) when is_binary(V) ->
 encode_value(V) ->
     term_to_binary(V).
 
--spec accept_value(string(), binary()) -> term().
+-spec accept_value(string(), riak_kv_wm_utils_dict(), binary()) -> term().
 %% @doc Accept the object value as a binary - content type can be used
 %%      to decode
-accept_value("application/x-erlang-binary",V) ->
-    binary_to_term(V);
-accept_value(_Ctype, V) ->
+accept_value("application/x-erlang-binary",MD, V) ->
```

Comment:
I was testing this incorrectly and have discovered some issues. I'll be updating the PR shortly with some changes.